packages/shell-chrome/views/_components/tab-link.edge

```diff
@@ -0,0 +1,13 @@
+<a
+    href="#"
+    title="{{ label || upperFirst(tab) }}"
+    :class="{
+        'text-white border-ice-500 cursor-default': activeTab === '{{ tab }}',
+        'border-transparent hover:text-white hover:border-ice-700': activeTab !== '{{ tab }}',
+    }"
+    class="px-4 border-b-3 xs:pl-2 xs:pr-3"
+    @click="activeTab = '{{ tab }}'"
+>
+    @!yield($slot.main)
+    <span class="hidden xs:inline-block">{{ label || upperFirst(tab) }}</span>
```

Comment:
we could pass this as part of the `$slot`, would mean we've got slightly more repetition (in the main template) but it'll be a bit more obvious how the tab links are generated and might make it easier to edit down the line (eg. you can search for the span with the right text content)

Reply:
Can do...I guess that'll make it easier to handle any dynamic content at an alpine level that might affect the nav.